app/views/forte_mailer/student_signup_notify_student.text.erb

```diff
@@ -4,5 +4,5 @@ Thank you for completing your registration.
 
 Please review the information below for accuracy. Please email any corrections or questions to TODO: Insert email here.
 
```

Comment:
I'm gonna use contact@forteacademy.org for now, which is their contact email on the current website